### PR TITLE
Add Deco Pro Small configuration

### PR DESCRIPTION
**Pull request by X9VoiD**
## Verification

[Discord](https://discord.com/channels/615607687467761684/789348845372178482/793353604861919232)

### CODE REVIEW

File: OpenTabletDriver/Configurations/XP-Pen/Deco Pro Small.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "Start": 80,
+        "StartInclusive": true,
+        "End": 96,
+        "EndInclusive": false
+      },
+      "VendorID": 10429,
+      "ProductID": 2313,
+      "InputReportLength": 10,
+      "OutputReportLength": 10,
+      "ReportParser": null,
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/XP-Pen/Deco Pro Small.json:20]: Use explicit report parser

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
very late review lol

File: OpenTabletDriver/Configurations/XP-Pen/Deco Pro Small.json (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver/Configurations/XP-Pen/Deco Pro Small.json]: Fixed

**Review by InfinityGhost — APPROVED** [review]
No auxiliary support, marking as missing features.

**Pull request merged**